dvc/utils/__init__.py

```diff
@@ -420,3 +420,8 @@ def resolve_output(inp, out):
     if os.path.isdir(out):
         return os.path.join(out, name)
     return out
+
+
+def is_exec_found(exec_name):
+    cmd = "({}) 2>{}".format(exec_name, os.devnull)
```

Comment:
No need for this to be in util, you only use it once. Move it to dvc/dagascii or, even better, inline it there and test `find_pager` as a whole, there is no good point in testing `os.system`.

Reply:
> there is no good reason to `os.system`.

how would you check if `less` exists?

Also, we discussed here
https://github.com/iterative/dvc/pull/2815#discussion_r352359765

> No need for this to be in util, you only use it once. Move it to dvc/dagascii or, even better, inline it there and test find_pager as a whole,

makes sense

Reply:
@xliiv sorry, that was a typo, I meant that there is no reason to test `os.system` specifically, which is what your tests for is_exec_found were doing.

Reply:
Also, any recommendations for this Deepsource error:
https://deepsource.io/gh/iterative/dvc/run/16d33e26-d89b-4055-8d7a-54e37e73ee32/#python

Reply:
@xliiv you can safely ignore that one. As noted in the PR template, DS and CC is advisory. 🙂

Reply:
> there is no good reason to test os.system.
btw, i was rather testing the command inside os.system, instead of os.system :)

Reply:
thanks and done

1. Replaced `is_exec_found` with inline in `dagascii.py`
2. added tests for `find_pager`

Reply:
should i resolved this comment?

Reply:
@xliiv sure, it is up to you, no need to ask. If you feel like this is resolved - just click "resolved" 🙂I also think it is resolved 🙂